app/controllers/concerns/sentry_controller_logging.rb

```diff
@@ -31,7 +31,7 @@ def user_context
 
   def tags_context
     { controller_name: }.tap do |tags|
-      if current_user.present?
+      if current_user.present? && current_user.respond_to?(:identity)
         tags[:sign_in_method] = current_user.identity.sign_in[:service_name]
```

Comment:
This is fine for now, overall we'll probably want to properly tag the sign in service name/account type though

Reply:
I forgot to circle back to this. I'll fix this